docs/university.md

```diff
@@ -2590,11 +2590,23 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 ### 研究生通知公告
 
-<Route author="mzr1996" example="/sjtu/gs/tzgg/pyxx" path="/sjtu/gs/tzgg/:type?" :paramsDesc="['默认列举所有通知公告']">
+<Route author="dzx-dzx" example="/sjtu/gs/enroll/59" path="/sjtu/gs/:type/:num?" :paramsDesc="['类别', '细分类别, 仅对`type`为`enroll`或`exchange`有效']">
```

Comment:
Do not remove the name of previous author(s)

Reply:
But…Wouldn't it be awkward that for every issue created for this route, the previous author will get pinged for the code he has no knowledge of?

Reply:
Pinging depends on the entries in `maintainer.js`.

Reply:
Oh OK. Thanks! It's now fixed in https://github.com/DIYgod/RSSHub/pull/12758/commits/b236b25f1f3b05e7ef3528c35a7612dc350590ec .